Python/compile.c

```diff
@@ -1714,6 +1716,12 @@ dict_lookup_arg(PyObject *dict, PyObject *name)
     return PyLong_AsLong(v);
 }
 
+static int
+compiler_lookup_cellvar(struct compiler *c, PyObject *name)
+{
+    return dict_lookup_arg(c->u->u_metadata.u_cellvars, name);
```

Comment:
Maybe add an `assert(c->u->u_metadata.u_cellvars);`? Technically, it shouldn't happen since there is a check when setting the attribute, but at least we're sure that we don't call this one before the `u_cellvars` is set.